packages/truffle-solidity-loader/lib/getTruffleConfig.js

```diff
@@ -3,12 +3,20 @@ const Logger = require('./logDecorator');
 
 const getTruffleConfig = function () {
   const isWin = /^win/.test(process.platform);
+  let truffleJsFile = findUp.sync('truffle.js');
+  let truffleConfigJsFile = findUp.sync('truffle-config.js');
   let file;
+  
+  if (truffleJsFile && truffleConfigJsFile && isWin) {
+    Logger.log("Warning: Both truffle.js and truffle-config.js were found. Using truffle-config.js");
+  } else if (truffleJsFile && truffleConfigJsFile && !isWin) {
+    Logger.log("Warning: Both truffle.js and truffle-config.js were found. Using truffle.js");
+  }
 
   if (isWin) {
```

Comment:
And here we just want to give precedence to `truffle-config.js` if they both exist. We are going to consider the Mac and Windows cases exactly the same.